packages/opentelemetry-semantic-conventions/src/resource/SemanticResourceAttributes.ts

```diff
@@ -480,6 +480,8 @@ As an alternative, consider setting `faas.id` as a span attribute instead.
 
 
 export enum CloudProviderValues {
+  /** Alibaba Cloud. */
+  ALIYUN = 'aliyun',
```

Comment:
This is a generated file. I guess the place to update this is in spec repo: https://github.com/open-telemetry/opentelemetry-specification/blob/main/semantic_conventions/resource/cloud.yaml

Reply:
for now you can just add const for that inside the package you are adding